cli/src/main/java/org/jboss/as/cli/embedded/EmbedServerHandler.java

```diff
@@ -267,9 +267,14 @@ public void cliEvent(CliEvent event, CommandContext ctx) {
     }
 
     private void configureLogContext(LogContext embeddedLogContext, File jbossHome, CommandContext ctx) {
-        File standaloneDir =  new File(jbossHome, "standalone");
-        File configDir =  new File(standaloneDir, "configuration");
-        File logDir =  new File(standaloneDir, "log");
+
+        final String serverBaseDir = WildFlySecurityManager.getPropertyPrivileged("jboss.server.base.dir", jbossHome.getAbsolutePath() + File.separator + "standalone");
+        final String serverConfigDir = WildFlySecurityManager.getPropertyPrivileged("jboss.server.configuration.dir", "configuration");
+        final String serverLogDir = WildFlySecurityManager.getPropertyPrivileged("jboss.server.log.dir", "log");
+
+        File standaloneDir =  new File(serverBaseDir);
+        File configDir =  new File(standaloneDir, serverConfigDir);
+        File logDir =  new File(standaloneDir, serverLogDir);
```

Comment:
This doesn't seem right. If the `jboss.server.log.dir` is set to something like `/var/log/wildfly` this will result in `$JBOSS_HOME/standalone/var/log/wildfly`.